README.md

```diff
@@ -71,6 +71,6 @@ There are tests for the C bindings in `gitoid/src/ffi/test.c` To exercise them:
 
 ```
 $ cargo build
-$ gcc --std=c11 -o c-test gitoid/src/ffi/test.c -L target/debug/ -lgitoid
+$ gcc --std=c11 -o c-test gitoid/test/c/test.c -L target/debug/ -lgitoid
```

Comment:
In the `Makefile`, the build command actually uses the `-I` method to add the `gitoid/include` directory to the include path, permitting the import in the test code to use `"gitoid.h"` directly, instead of needing to specify a relative path.

I personally like that approach better, as it makes the problem of telling the compiler where to find the header the issue of the caller of the build, rather than having a path relationship between the test file and header file encoded in the test file itself.

Happy to discuss though!

Reply:
Ah! Forgot about that - fixing!